### PR TITLE
Add posichain mainnet

### PR DESCRIPTION
**Pull request by dannyposi**
Changes proposed in this pull request:
- Add Posichain Mainnet

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -1795,5 +1795,21 @@
...
     "start": 1126886,
     "logo": "ipfs://QmaARJiAQUn4Z6wG8GLEry3kTeBB3k6RfHzSZU9SPhBgcG"
+  },
+  "900000": {
+    "key": "900000",
+    "name": "Posichain Mainnet",
+    "shortName": "PSC",
+    "chainId": 900000,
+    "network": "mainnet",
+    "multicall": "0xbe779BaA9dC15Ef307bdf4dDb35440A0a421a89A",
```
Comment (ChaituVR) [inline src/networks.json:1805]: Is there a way to check the code here https://ubiqscan.io/address/0x6668750957e4083725926b71c41bdf1434c73262

Everything else works good

Comment (dannyposi) [reply src/networks.json:1805]: Why we need to check that code? about multicall contract, you can check here: https://explorer.posichain.org/address/0xbe779BaA9dC15Ef307bdf4dDb35440A0a421a89A?activeTab=7

Comment (dannyposi) [reply src/networks.json:1805]: @ChaituVR please help to check

Comment (ChaituVR) [reply src/networks.json:1805]: Oops wrong link paste, Thanks @dannyposi

File: src/networks.json [hunk 2]
```diff
@@ -1795,5 +1795,21 @@
...
+    "rpc": [
+      "https://api-a001.s0.posichain.org"
+    ],
+    "explorer": {
+      "url": "https://explorer.posichain.org"
+    },
+    "start": 3860000,
```
Comment (ChaituVR) [inline src/networks.json:1812]:
```suggestion
    "start": 2560501,
```

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**